### ISSUE TITLE
Cannot connect to runtine process, timeout after 10000ms - (reason: Cannot connect to the target: connect ECONNREFUSED 127.0.0.1=9229

### ISSUE THREAD

**Issue by Kelvijn** [Microsoft/vscode#64069]
```js
{
  "version": "0.2.0",
  "configurations": [{
    "type": "node",
    "request": "launch",
    "name": "Launch via NPM",
    "runtimeExecutable": "npm",
    "runtimeArgs": [
      "run-script",
      "debug"
    ],
    "port": 9229
  }]
}
```

Debugger worked fine before, reinstalled vscode and extensions. Restarted pc. Can't find a solution.
Just like any other day for the past couple of weeks I was developing a Loopback 4 app. Sometimes i would get this error but restarting my pc always worked. 
How to reproduce: Debug -> start debugging 

**OS**
windows 10 home

**VSCODE VERSION**
C:\Users\Calvin Swinnen>code --version
1.29.1
bc24f98b5f70467bc689abf41cc5550ca637088e
x64

**INSTALLED EXTENSIONS**
C:\Users\Calvin Swinnen>code --list-extensions
dracula-theme.theme-dracula
Equinusocio.vsc-material-theme
HookyQR.beautify
Mikael.Angular-BeastCode
robertohuertasm.vscode-icons
zhuangtongfa.Material-theme

**package.json scripts**
    "prestart": "npm run build",
    "start": "node .",
    "predebug": "npm run build",
    "debug": "node --nolazy --inspect-brk=9229 ."

**Comment by weinand** [comment]
Please see [this guide](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md#writing-good-bug-reports-and-feature-requests) for how to file a proper issue. Include: 

* VS Code version
* OS version
* Installed extensions
* Steps to reproduce

How does your "debug" npm script look like?

**Comment by Kelvijn** [comment]
I have updated my comment!

**Comment by weinand** [comment]
Btw, if you use VS Code's "Help > Report Issue" command, you get all that information automatically entered in an github issue.

**Comment by roblourens** [comment]
Why are you hardcoding the port? Maybe there is something else on that port.

**Comment by weinand** [comment]
Adding the port to the launch config prevents node-debug from adding a "--inspect-brk" to the "npm" command (which would not work; see https://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script).

**Comment by Kelvijn** [comment]
Hi, sorry for the late reply.
With my configuration the debugger starts 40% of the time. 
If I remove the port from the configuration I always get the connection error.
I got this configuration from https://code.visualstudio.com/docs/nodejs/nodejs-debugging

I tried your config:
`"prestart": "npm run build",
    "start": "node .",
    "predebug": "npm run build",
    "debug": "node --inspect-brk ."`
`"configurations": [{
    "type": "node",
    "request": "launch",
    "name": "Launch via NPM",
    "runtimeExecutable": "npm",
    "runtimeArgs": [
      "run-script",
      "debug",
      "--"
    ]
  }]`

result: it gives the same error 100% of the time with a random port...
